samples/subsys/ipc/openamp/README.md

```diff
@@ -0,0 +1,72 @@
+OpenAMP Example Application
```

Comment:
This file should be called README.rst (with an .rst extension)

Add a line before this with a document label to make it easier to reference externaly:

    .. _openAMP_example: